content/presubmission_inquiry.md

```diff
@@ -38,11 +37,11 @@ inquiry, the following rules have been proposed:
 1. Lorem ipsum
 1. Lorem ipsum
 
-These rules are chosen so that readers of varying experience levels will be able to gain from the article and range from the high level (Should I use deep learning for this problem?) to implementation best practices.
+These rules have been devised to effectively reach audiences of varying DL expertise readers ranging from high level guidance to implementation of best practices.
 
-Upon determination that this manuscript will be suitable for submission to PLOS Computational Biology, I will further engage with the community to solicit feedback and contributions. All authors of the paper will meet the ICMJE authorship standards.
+Upon notification that this manuscript will be suitable for submission to PLOS Computational Biology, I will further engage with the community to solicit feedback and contributions. All authors of the paper will meet the ICMJE authorship standards.
 
-By providing guidance on deep learning, powerful DL methods can be properly utilized by the community towards a deeper understanding of biology. Here, we aim to increase the number of biological researchers that take advanatage of DL methods (by making DL more accessible) and improve the overall the quality and reproducibility of the research (by helping researchers avoid common mistakes).
+By providing guidance on deep learning, powerful DL methods can be properly utilized by the biological community towards a deeper understanding of biology. Here, we aim to increase the number of biological researchers that take advanatage of DL methods (by making DL more accessible) and improve the overall quality and reproducibility of DL research by helping researchers avoid common pitfalls.
```

Comment:
Some typos. Would revise to: 
"By providing guidance on deep learning, powerful DL methods can be more properly utilized by the biological community. Here, we aim to increase the accessibility of DL techniques to biology and thereby improve the overall quality and reproducibility of DL in the literature."

Reply:
Should we also switch the use of "deep learning" to "DL" to make it more consistent?

Reply:
Should we also switch from "deep learning" to "DL" to make this more consistent?

Reply:
Maybe i am to nit-picky but "biological researchers" sounds weird ;). Maybe we can just say "biologists" or "computational as well as experimental biologists"